src/ResourceManager/Compute/Commands.Compute/Extension/DSC/SetAzureVMDscExtensionCommand.cs

```diff
@@ -460,20 +460,6 @@ private ConfigurationUris UploadConfigurationDataToBlob()
                     configurationDataBlobReference.Uri.AbsoluteUri,
                     () =>
                     {
-                        if (!Force && configurationDataBlobReference.Exists())
```

Comment:
removing force here would mean that we by default always update the storage blob - no warning to the user